### ISSUE TITLE
"Modernizing" Heron styling

### ISSUE THREAD

**Issue by dracic** [heron-mc/heron-mc#445]
Any HTML/CSS guru willing to make Heron appearance less Windows'ish?

Good Heron examples (really like this one, but with some changes):

http://us1.mapcentia.com/apps/mapclient/mydb/ny

I will try something between grey theme and this one...

Good GeoExt (GeoMapFish) examples:

http://geo.yverdon.ch/theme/plan_de_ville

**Comment by justb4** [comment]
Good idea, I especially like: http://geo.yverdon.ch/theme/plan_de_ville. I also get bored with the Blue/Windows style. 

Mostly I use the ``gray`` neutral style as here: http://kadviewer.kademo.nl. This theme is available in the Heron distro under ``heron/resources/css`` as ``default-theme-gray.css``. There is ``blue``, ``grey and ``greenery``.

Specific themes like http://lib.heron-mc.org/heron/latest/examples/theming were built (CSS generated) using an online ExtJS v3 Theme Builder but it is not available anymore: http://extbuilder.dynalias.com

More theming possibilities would be possible by upgrading to GeoExt2 with ExtJS 4 but this is a major undertaking...

**Comment by dracic** [comment]
I like colors from second GMF example, but flat controls from Mapcentia, without blue buttons (button text is not clearly visible when disabled)...
:-)

I think uprading to Geoext2 doesn't make any sense since Geoext3 is allready work in progress but is obvious that the trends in web mapping (even at Boundless) to replace gxp with some Angular or Bootstrap + OL3 + Cesium responsive combination.

For example, Camtptocamp did a lot of work with Geoext, gxp, Mapfish, Geomapfish, but they are moving to:

https://groups.google.com/forum/#!topic/geomapfish/tIUCl9_Z16k

**Comment by dracic** [comment]
http://sourceforge.net/projects/extremeextjsbld/

**Comment by el486** [comment]
Hi, dracic. I taked your idea and modify (or un-modify) the mapcentia template using the gray theme as base. You can see the result in my test site:

http://www.mosp.gba.gov.ar/sig_hidraulica/ms/heron/visor105/

the template uses all the images from the gray theme, no other resources needed.

[xtheme-gray-flat.css.txt](https://github.com/heron-mc/heron-mc/files/103294/xtheme-gray-flat.css.txt)

**Comment by dracic** [comment]
Nice job...

Cartoview has a nice white theme:
http://www.cartoview.org/maps/89
